identifiers/country-es/provincia.csv

```diff
@@ -0,0 +1,50 @@
+ocd-division/country:es/state:ga/cd:a_coruña,A Coruña
```

Comment:
Similarly because of #170 something like "province:" instead of "cd:" makes more sense. @jpmckinney

Reply:
don't have a problem with province instead of cd. would be okay with that.
But should i use then 
- ocd-division/country:es/state:ga/province:a_coruña,A Coruña
OR with hasc code (because it isnt a cd)?
- ocd-division/country:es/state:ga/province:ac,A Coruña

Reply:
If there's an official set of abbreviations (either ISO or HSAC) for provinces, then use that. Otherwise the full names are good.

Also, could we sort these by OCD-ID instead of name? It's easier to look it over and see how many provinces are in each state.

Reply:
Yes, there is. http://www.statoids.com/yes.html
ocd-division/country:es/state:ga/province:a_coruña,A Coruña
HASC:  ES.GA.**AC**

And yes, i can sort by ocd-id. should be no problem.

Reply:
cd changes to province with iso code
@jdmgoogle @jpmckinney